git-pr-chain.py

```diff
@@ -710,6 +774,12 @@ def main():
     sp_log = subparser.add_parser("log", help="List commits in chain")
     sp_log.set_defaults(func=cmd_log)
 
+    sp_new = subparser.add_parser("new", help="Mark HEAD as starting a new PR in the chain")
+    sp_new.set_defaults(func=cmd_new_pr)
+
+    sp_end = subparser.add_parser("end-chain", help="Add a commit to mark the end of the chain")
```

Comment:
Please run `black` before submitting!

Reply:
What does running "black" mean?

Reply:
Found it. A code formatting tool for python! 😮